### ISSUE TITLE
Fix spacing/alignment of legend in ROBINS-I `rob_summary()` templates

### ISSUE THREAD

**Issue by mcguinlu** [mcguinlu/robvis#41]
![ROBINS-I (24)](https://user-images.githubusercontent.com/41202585/63082766-fe7d1a80-bf3e-11e9-8a85-c31a9edca255.png)

Centre legend keys and text in the legend box, the same as in the other (ROB2/ROB1/QUADAS-2 tools)

**Comment by daniloimparato** [comment]
Hey @mcguinlu, I'd like to take a look at this issue.

Upon running `rob_summary(data = data_rob1, tool='ROBINS-I')` I get the following plot:
![rob_summary](https://user-images.githubusercontent.com/13784115/67640058-5d172b00-f8d6-11e9-8c2b-51295e0b4f6b.png)

I this the expected behavior? If so, could you elaborate a bit more on what the spacing should look like?

**Comment by mcguinlu** [comment]
Hi @daniloimparato, 

Thanks for expressing your interest in this. The first thing to note is that the example datasets are designed to be used only with the template contained in their name, i.e. if you are using `data_rob1`, then you need to specifiy ` tool= "ROB1" `, which is why your image above has so many NAs.

To elaborate on the original issue, the problem is that the coloured boxes are not vertically centered within the legend box when using the ROBINS-I template (`data = data_robins, tool='ROBINS-I'`). I've added red arrows to the box in the images below to help demonstrate this.

The ROBINS-I template currently gives the following:

`rob_summary(data = data_robins, tool='ROBINS-I')`
![ROBINS-I](https://user-images.githubusercontent.com/41202585/67694167-d37d6100-f99a-11e9-8683-a79ea399bea4.png)


For an example of what it should look like, with the legend keys correctly vertically centered:

`rob_summary(data = data_rob2, tool='ROB2')`
![ROB2](https://user-images.githubusercontent.com/41202585/67694164-d24c3400-f99a-11e9-8869-af4e600739d7.png)

I haven't been able to work out what is causing this, so if you can spot the error, feel free to open a PR with a fix.

Thanks in advance!

**Comment by daniloimparato** [comment]
My bad, I completely missed the `data_robins` dataset.

However, I couldn't reproduce the misalignment. What system are you on?

**Comment by mcguinlu** [comment]
Hi @daniloimparato, 

Very sorry, I (obviously) completely missed your response to this. As it turns out, I inadvertedly found the cause of this issue earlier today when tidying up some of the code. 

It turns out that I had left the following code in the ROBINS-I template from when I was considering having a title in the legend box:

```
ggplot2::guides(
    fill = ggplot2::guide_legend(title = "Risk of bias judgement",
                                                  title.position = "bottom",	
                                                  title.hjust = 0.5,	
                                                  reverse = TRUE )
                         )
```

This was causing the legend keys/text to be shifted vertically. Once I replaced the above code with the following, the issue was fixed (see image below):
```
ggplot2::guides(fill = ggplot2::guide_legend(reverse = TRUE))
```

![ROBINS-I (30)](https://user-images.githubusercontent.com/41202585/70807520-9b4ca900-1db5-11ea-9804-fc7ec931c859.png)

Sorry again for not getting back to you on this, and thanks for your interest in helping out. Do feel free to give any of the other issues a go if they take your fancy! 

Luke
